google-http-client-apache-v5/src/main/resources/META-INF/native-image/com.google.http-client/google-http-client-apache-v2/reflect-config.json

```diff
@@ -0,0 +1,59 @@
+[
```

Comment:
Do we know these entries are correct for the v5 implementation, or are we just copying from the v4 `native.config`?

Reply:
I tried running `mvn -Pnative native:compile`

It failed because we don't have a test config ([logs](https://paste.googleplex.com/4939824807280640))

Reply:
> Test configuration file wasn't found. Make sure that test execution wasn't skipped.

Try `mvn verify -Pnative`.

Reply:
[Same situation](https://paste.googleplex.com/6306119184285696?raw)

Reply:
`-Pnative-tests` seems to work.